arctic/_cache.py

```diff
@@ -61,7 +61,8 @@ def append(self, key, append_data):
         try:
             self._cachecol.update_one(
                 {'type': key},
-                {'$push': {'data': append_data}}
+                {'$push': {'data': append_data}},
```

Comment:
Do you need setOnInsert also for date and type?

https://stackoverflow.com/a/47004771/1542667

Reply:
Also remove the comment on line 60